exchanges/order/order_test.go

```diff
@@ -934,6 +934,26 @@ func TestUpdateOrderFromModifyResponse(t *testing.T) {
 	assert.Nil(t, od.Trades)
 }
 
+func TestTimeInForceIs(t *testing.T) {
+	t.Parallel()
+	tifValuesMap := map[TimeInForce][]TimeInForce{
+		GoodTillCancel | PostOnly:    {GoodTillCancel, PostOnly},
+		GoodTillCancel:               {GoodTillCancel},
+		GoodTillCrossing | PostOnly:  {GoodTillCrossing, PostOnly},
+		ImmediateOrCancel | PostOnly: {ImmediateOrCancel, PostOnly},
+		GoodTillDay:                  {GoodTillDay},
+		FillOrKill | PostOnly:        {FillOrKill, PostOnly},
```

Comment:
Fill Or kill is usually taker only, I haven't seen exchanges handle a limit order allowance that must be completely filled. So I think Post only can be removed.